Makefile

```diff
@@ -26,45 +27,52 @@ default: help
 build-quick: ## Build the image 🚀(quicker).
 	make build options=""
 
-build: ## Build all the images 🚀
-build: build-5.6 build-7.0 build-7.1
-tag: ## Tag all the images
-tag: tag-5.6 tag-7.0 tag-7.1
-test: ## Test all the imges
-test: test-5.6 test-7.0 test-7.1
-push: ## Push all the images to docker hub
-push: push-5.6 push-7.0 push-7.1
-
-build-%: options ?=--no-cache --pull
-build-%: ## Build an individual image 🚀
-	${DOCKER} build ${BUILD_ARGS} ${options} --tag ${DOCKER_REPOSITORY}:php-$* ./php-$*
-
-test-%: ## Test the images.
-	${BATS} ./tests/graze_composer_php-$*.bats
-
-deploy-%: ## Deploy a specific version
-	make tag-$* push-$*
-
-tag-5.6:
-	${DOCKER} tag ${DOCKER_REPOSITORY}:php-5.6 ${DOCKER_REPOSITORY}:php-5
-
-push-5.6:
-	${DOCKER} push ${DOCKER_REPOSITORY}:php-5.6
-	${DOCKER} push ${DOCKER_REPOSITORY}:php-5
-
-tag-7.0: ## Nothing to do
-
-push-7.0:
-	${DOCKER} push ${DOCKER_REPOSITORY}:php-7.0
-
-tag-7.1:
-	${DOCKER} tag ${DOCKER_REPOSITORY}:php-7.1 ${DOCKER_REPOSITORY}:php-7
-	${DOCKER} tag ${DOCKER_REPOSITORY}:php-7.1 ${DOCKER_REPOSITORY}:latest
-
-push-7.1:
-	${DOCKER} push ${DOCKER_REPOSITORY}:php-7.1
-	${DOCKER} push ${DOCKER_REPOSITORY}:php-7
+all-%:
+	make php-$* COMPOSER_VER=1.4.1
+	make php-$* COMPOSER_VER=1.4.0
+	make php-$* COMPOSER_VER=1.3.3
+	make php-$* COMPOSER_VER=1.3.2
+	make php-$* COMPOSER_VER=1.3.1
+	make php-$* COMPOSER_VER=1.3.0
+
+php-%:
+	make $* PHP_VER=7.1
+	make $* PHP_VER=7.0
+	make $* PHP_VER=5.6
```

Comment:
Guess the readme might need more text, the $* is not for versions but for the command to perform (`build`/`test`/`tag`/`push`)

so if you do `make build` it will call `all-build` which will call `php-build` with different composer versions.